aws_lambda_builders/workflows/nodejs_npm/workflow.py

```diff
@@ -75,39 +81,82 @@ def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, runtim
         ]
 
         if self.download_dependencies:
-            # install the dependencies into artifact folder
             self.actions.append(
-                NodejsNpmWorkflow.get_install_action(source_dir, artifacts_dir, subprocess_npm, osutils, self.options)
+                NodejsNpmWorkflow.get_install_action(
+                    source_dir=source_dir,
+                    install_dir=self.build_dir,
+                    subprocess_npm=subprocess_npm,
+                    osutils=osutils,
+                    build_options=self.options,
+                    install_links=self.build_dir == self.source_dir,
```

Comment:
We want to use install-links so npm installs local dependencies without any symbolic links and without dev dependencies or temporary files, but instead just like any other dependency.